src/servers/healthz-handler.ts

```diff
@@ -0,0 +1,44 @@
+import express from "express"
```

Comment:
should this be in `src/servers/`? it's not meant to be run independantly? I think this should be somewhere else (not sure where)

Reply:
I can rename `src/servers/graphql-middlewares` to `src/servers/middlewares` and move it to that folder. This is another middleware implementation. what do you think?

Reply:
does this matters if this files is included in this pattern? https://github.com/GaloyMoney/galoy/blob/a9cdea18a037720d749657c879012b8dc4c3b756/tsconfig-build.json#L3

and is it included in this pattern if we add it in a middlewares folder?

Reply:
`**` should take care of inner folders, it is safe to move it. let me know if is necessary